gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/ApisResource.java

```diff
@@ -261,6 +261,9 @@ public ApiListItemPagedResult getApis(
     @Permissions({ @Permission(value = RolePermission.ENVIRONMENT_API, acls = RolePermissionAction.CREATE) })
     public Response createApi(@Parameter(name = "api", required = true) @Valid @NotNull final NewApiEntity newApiEntity)
         throws ApiAlreadyExistsException {
+        if (DefinitionVersion.valueOfLabel(newApiEntity.getGraviteeDefinitionVersion()).equals(DefinitionVersion.V1)) {
```

Comment:
Shouldn't we add this check in the service instead of the REST resource in order to force this behavior and throw an exception when the service is used elsewhere; for example by plugins?